xml/ha_rear.xml

```diff
@@ -24,7 +24,7 @@
       <abstract>
         <para>
     &rear.long; (formerly <quote>ReaR</quote>, in this chapter
-    abbreviated as &rear;) is a disaster recovery framework for use by
```

Comment:
I would like to have the word 'formerly' removed
because the abbreviation 'ReaR' is still used
and it is the officially right one, cf.
https://github.com/rear/rear/issues/1033